scripts/get_redash_api_key.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Check if the class name was provided as an argument
+if [ -z "$1" ]; then
+  echo "Class name not provided. Exiting..."

```

Comment:
do we need this condition?

Reply:
just given to see if classname provided.